docs/UserGuide.md

```diff
@@ -323,6 +323,21 @@ Action     | Format, Examples
 
 ### Name
 
+Format: `name` where `name` is a sequence of characters corresponding to a student's name
+
+Constraints:
+* `name` can contain any alphanumeric character or whitespace.
+* `name` can contain hyphens, commas and apostrophes.
+* `name` can be 1 to 300 characters long.
+  
```

Comment:
shall we also 'add `name` can not be blank', ignore if you think it's not necessary